src/pages/Lending/hooks/useLendingPositionData.tsx

```diff
@@ -19,7 +19,8 @@ export const useLendingPositionData = ({ accountId, assetId }: UseLendingPositio
   const poolAssetMarketData = useAppSelector(state => selectMarketDataById(state, assetId))
 
   const lendingPositionData = useQuery({
-    staleTime: Infinity,
+    // 2 minutes before the data is considered stale, meaning firing this query will trigger queryFn
```

Comment:
does trk query use stable stringifcation of keys?
`lendingPositionQueryKey` contains a json object that doesnt guarantee key ordering, so if not we should change the key to be stable, e.g 
instead of
`['thorchainLendingPosition', { accountId, assetId }]`
do 
`['thorchainLendingPosition', accountId, assetId]`

Reply:
Yes it does! https://stackoverflow.com/a/73342066